documentation/configuring_bolt.md

```diff
@@ -1,59 +1,190 @@
 # Configuring Bolt
 
 You can configure Bolt's options and features at a project level, a user level,
-or a system-wide level. 
+or a system-wide level.
 
-Some options and features you can configure include:
```

Comment:
I really like the layout of this file - it's clear, well structured, and easy to follow